xml/soc-operations-maintenance.xml

```diff
@@ -31,8 +31,11 @@
   <note>
    <title>No Maintenance Mode</title>
    <para>
-    As of &productname; 8, it is not possible to put
-    &cloud; into <quote>Maintenance Mode</quote>.
+    As of &productname; 8, it is not possible to put your entire
+    &cloud; into <quote>Maintenance Mode</quote>, as operations at this level
+    are <literal>read-only</literal>. Services and elements must be put into
```

Comment:
Operations in OpenStack are not read-only so this is not quite right, but I can see where the misunderstanding came from.  In https://bugzilla.suse.com/show_bug.cgi?id=1108818#c0 I wrote:

> My best guess is that it refers to the fact that OpenStack *itself* has no concept of maintenance mode (e.g. allowing only read-only operations on the cloud state / underlying database).

The bit in parentheses was an example of what a maintenance mode in OpenStack *might* provide if it was implemented in the future, not a description of the status quo.

Here is my suggestion for this paragraph:

```xml
It is not possible to put your entire &cloud; into <quote>Maintenance
Mode</quote> (e.g. limiting all users to read-only operations on the
control plane), as OpenStack does not yet support this.  However when
Pacemaker is deployed to manage HA clusters, it should be used to place
the services and cluster nodes it owns into <quote>Maintenance Mode</quote>
before performing maintenance functions on them.  See [INSERT LINK
TO RELEVANT SECTION OF HA GUIDE] for more details on how to do this.
```